docs/openshift/deploy.md

```diff
@@ -0,0 +1,113 @@
+# Deploy an OpenShift Cluster
+
+## Install Pre-requisites
+
+All the commands in this guide require both the Azure CLI and `acs-engine`. Follow the [installation instructions to download acs-engine before continuing](../acsengine.md#install-acs-engine) or [compile it from source](../acsengine.md#build-from-source).
+
+To install the Azure CLI, follow [the official documentation](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli) for your operating system.
+
+## Overview
+
+[acs-engine](https://github.com/Azure/acs-engine/blob/master/docs/acsengine.md) reads a cluster definition (or api model) which describes the size, shape, and configuration of your cluster. This guide takes the default configuration of one master and two Linux nodes, where one node is used by the OpenShift internal infrastructure, and the other one is for the end-user workloads (compute node). The nodes are broken into two different agent pool profiles in the definition. If you would like to change this configuration, edit [examples/openshift.json](/examples/openshift.json) before continuing.
```

Comment:
s/takes/follows/

Reply:
s/the end-user/end-user/

Reply:
> The nodes are broken into two different agent pool profiles in the definition. If you would like to change this configuration,

At least one of each node type is required for a working OpenShift cluster.  In the openshift.json file, one agent pool specifies the number of infrastructure node(s); another is used to specify the number of compute node(s).  If you would like to change these numbers,